djangoevents/domain.py

```diff
@@ -47,8 +71,43 @@ class BaseEntity(BaseAggregate):
 
     OBSOLETE! Interface kept for backward compatibility.
     """
+    @classmethod
+    def is_abstract_class(cls):
+        return super().is_abstract_class() or cls is BaseEntity
 
     @classmethod
     def mutate(cls, entity=None, event=None):
         warnings.warn("`BaseEntity` is depreciated. Please switch to: `BaseAggregate`", DeprecationWarning)
         return super().mutate(aggregate=entity, event=event)
+
+
+def _list_subclasses(cls):
+    """
+    Recursively lists all subclasses of `cls`.
+    """
+    return cls.__subclasses__() + [g for s in cls.__subclasses__() for g in _list_subclasses(s)]
```

Comment:
Yeah, so it works, but the results of this function might be misleading - I'm thinking if not to remove them here? But it's a minor thing.